example_config.yaml

```diff
@@ -1,8 +1,10 @@
 # device id is unique to each device and is used on the cloud
 device_id: "uid8933674632"
 
-ip_access_point: "127.80.034"
+access_point_ssid: "127.80.034"
```

Comment:
ssid will be the name of the wifi network and not an ip. Can we change the placeholder value to avoid confusion?

Reply:
Done